toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/components/datetime/DateTimeField.kt

```diff
@@ -64,12 +67,15 @@ internal fun DateTimeField(
     val isEditable by state.isEditable
     val isRequired by state.isRequired
     val epochMillis by state.value
-    val pickerStyle = if (state.shouldShowTime) {
+    val shouldShowTime = rememberSaveable {
```

Comment:
```suggestion
    val shouldShowTime = rememberSaveable {
```
Does this need to be remembered?

Reply:
nope. at one point it seemed necessary, but not now.